packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-glossary.tsx

```diff
@@ -5,16 +5,20 @@ import { Localize } from '@deriv/translations';
 
 const ContractTypeGlossary = ({
     category,
+    is_vanilla_fx = false,
     is_major_pairs = false,
     is_multiplier_fx = false,
 }: {
     category: string;
+    is_vanilla_fx?: boolean;
     is_major_pairs?: boolean;
     is_multiplier_fx?: boolean;
 }) => {
     let content;
+    let trade_category = category;
+    if (trade_category === 'vanillalongcall' && is_vanilla_fx) trade_category = VANILLALONG.FX;
```

Comment:
I guess it can be written like 
`let trade_category = category  === VANILLALONG.CALL && is_vanilla_fx ? VANILLALONG.FX : category`
But your option has better readability